pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationpolicy_types_test.go

```diff
@@ -0,0 +1,135 @@
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("Node info list", func() {
+	Context("is empty", func() {
+		originalNodeInfoList := NodeInfoList{}
```

Comment:
For "is empty" having an originalNodeInfoList is overkill

Reply:
Done